compiler/AST/UseStmt.cpp

```diff
@@ -150,6 +155,11 @@ bool UseStmt::isARename(const char* name) const {
   return renamed.count(name) == 1;
 }
 
+// Specifically for when the module being used is renamed
+bool UseStmt::isARename() const {
+  return strcmp(modRename, "") != 0;
```

Comment:
I'd normally write this as `return modRename[0] != '\0';`

Reply:
Also I would consider:
 * storing the same module name in modRename if no renaming has occured
 * always storing an `astr` in modRename